static/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
```

Comment:
Probably should do the `UTF-8` in lowercase for consistency and to follow the style guide.